src/main/java/de/hsrm/vegetables/my_food_coop_service/services/DisposeService.java

```diff
@@ -0,0 +1,52 @@
+package de.hsrm.vegetables.my_food_coop_service.services;
+
+import de.hsrm.vegetables.my_food_coop_service.model.DisposedItem;
+import de.hsrm.vegetables.my_food_coop_service.model.VatDetailItem;
+import de.hsrm.vegetables.my_food_coop_service.repositories.DisposedRepository;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__({@Autowired}))
+public class DisposeService {
```

Comment:
I'd call this Dispose**d**Service, as the DTOs are called Dispose**d**Dto as well